lib/sms_confirm.rb

```diff
@@ -0,0 +1,18 @@
+require("bundler")
+Bundler.require()
+class SMS
+
+  def intitializ
+    account_sid = "ACa807f58093c9262c30d6551be355db48"
+    auth_token = "ad862830a9ba96f9b5e3031c73eb4d94"
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+  end
+
+  def send_confirmation
+    @client.api.account.messages.create(
+      to: "447561848422",
+      from: "441725762059",
+      body: "Order confirmed! It will arrive by #{(Time.new + 3600).strftime("%-I:%M%P")}"
```

Comment:
Echoing above comment - Consider taking "%-I:%M%P" out and setting it equal to a variable at the start of your program, for example TIME_FORMAT = "%-I:%M%P"

Then at the end of your file use ".strftime(TIME_FORMAT) - this will remove the 'magic number'-ish style of the current program. Also consider changing the format to "%H:%M" which will do a similar thing to yours, expect will be a 24 hour clock which removes the need for %P (AM/PM)